### ISSUE TITLE
Ventoy 1.0.07, 1.0.10 working on newer PC but not on older PCs

### ISSUE THREAD

**Issue by georgoswonkos** [ventoy/Ventoy#128]
I tested the same Ventoy 1.0.07 USB stick with the ISO "ubuntu-19.10-desktop-amd64.iso" on old and somewhat newer PCs. The newer worked flawlessly, the 2 older PCs froze after selecting the ISO in Ventoy and pressing "ENTER". 

All are 64-bit CPUs and work with this ISO when I put it directly on a USB stick without Ventoy. So it is not that Ventoy doesn't work with a certain ISO distribution, It rather seems that Ventoy has got a problem with certain (maybe non-UEFI) hardware.

Athlon A4-5300 CPU, Board: MSI FM2-A75MA-P33 (supports UEFI) - works ok
Athlon II X2 255 CPU, Board MSI 770 – C45, no UEFI - freeze
Athlon 3500 CPU, Board Asus A8N5X, no UEFI - freeze

I just tested Ventoy 1.0.10 on PC #2 and still get the freeze. I also used different ISOs for testing with Ventoy 1.0.07. Same result. PC #1 works, PCs #2 and #3 freeze.

**Comment by ventoy** [comment]
What about the latest release?

**Comment by georgoswonkos** [comment]
I did a new test with Ventoy 1.0.27, various ISOs and documented it:
1. Athlon A4-5300 CPU, Board: MSI FM2-A75MA-P33 (supports UEFI):
Linux Mint 20 x64 - Ventoy ok
Opensuse 15.2 x64 - Ventoy ok
Windows 10 20H2 x64 - Ventoy ok
2. Athlon II X2 255 CPU, Board MSI 770 – C45, no UEFI
Linux Mint 20 x64 - Ventoy ok
Opensuse 15.2 x64 - Ventoy ok
Windows 10 20H2 x64 - Ventoy ok
3. Athlon II X4 645 CPU, Board MSI 770 – C45, no UEFI
Linux Mint 20 x64 - Ventoy ok
Opensuse 15.2 x64 - Ventoy ok
Windows 10 20H2 x64 - Ventoy ok
4. Athlon 3500 CPU, Board Asus A8N5X, no UEFI, very old -> did not work:
Linux Mint 20 x64 - Ventoy - error during boot: Unable to find a medium containing a live file system...
Opensuse 15.2 x64 - Ventoy - error during boot: Unable to find a medium containing a live file system...
  However the Athlon 3500 worked when I used a USB stick with an ISO written directly on it without Ventoy:
Linux Mint 20 x64 - native Usb stick ok
Opensuse 15.2 x64 - native Usb stick ok
